### PR TITLE
website/docs: Add links

### PR DESCRIPTION
**Pull request by tanberry**
Add links to new separate pages for each web server

### CODE REVIEW

File: website/docs/providers/proxy/forward_auth.mdx [hunk 1]
```diff
@@ -23,8 +29,8 @@ This mode differs from the _Forward auth (single application)_ mode in the follo
 
 There are however also some downsides, mainly the fact that you **can't** restrict individual applications to different users.
 
-The only configuration difference between single application and domain level is the host you specify.
-
-For single application, you'd use the domain which the application is running on, and only `/outpost.goauthentik.io` is redirected to the outpost.
-
-For domain level, you'd use the same domain as authentik.
+For more information and configuration templates for each web server, refer to the following:
```
Comment (BeryJu) [inline website/docs/providers/proxy/forward_auth.mdx:32]: This part can be automated using this, which as the advantage of not needing to maintain the list:

```suggestion
import DocCardList from "@theme/DocCardList";
import { useCurrentSidebarCategory } from "@docusaurus/theme-common";

<DocCardList items={useCurrentSidebarCategory().items} />
```

File: website/docs/providers/proxy/forward_auth.mdx [hunk 2]
```diff
@@ -23,8 +29,8 @@ This mode differs from the _Forward auth (single application)_ mode in the follo
 
 There are however also some downsides, mainly the fact that you **can't** restrict individual applications to different users.
 
-The only configuration difference between single application and domain level is the host you specify.
```
Comment (BeryJu) [inline website/docs/providers/proxy/forward_auth.mdx:26]: I don't think this part should be removed as it still applies and is useful to people that can't use an existing config, maybe it needs some rewording though

**Review by tanberry — COMMENTED** [review]
Still WIP

File: website/docs/providers/proxy/forward_auth.mdx (hunk 2 above)
Comment (tanberry) [reply website/docs/providers/proxy/forward_auth.mdx:26]: Oooof no i agree, I am not trying to remove it, was trying to move it up above, but am making a bit of a mess with this PR. Gimme a minute... ;-)

File: website/docs/providers/proxy/forward_auth.mdx (hunk 1 above)
Comment (tanberry) [reply website/docs/providers/proxy/forward_auth.mdx:32]: Oh, that would be great to automate this.

File: website/docs/providers/proxy/forward_auth.mdx (hunk 2 above)
Comment (tanberry) [reply website/docs/providers/proxy/forward_auth.mdx:26]: I'd like to either go ahead and merge this PR as is, then keep working on the page, or close this PR
 and start over. Will for sure use your automated process for listing the linked docs.

**Pull request closed without merging**